From 09039796583d573977f492c498eca4c7f1573e27 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 10 Sep 2018 15:40:16 +0800
Subject: [PATCH 4571/5242] MLK-19515 soc: imx: ipc: remove unnecessary set
 wake for mu

commit  c88f191a62066025a32e84be34c0dff739501291 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since MU now is NOT in WU irq domain, so there is no irq_set_wake
callback available, below message will come out during kernel boot
up:

imx8mu_init: set_irq_wake failed: -6

GIC/MU now are powered off during suspend, so it is unnecessary to
call irq_set_irq_wake() for MU, we can remove it to avoid the failure
message.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/sc/main/ipc.c |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/soc/imx/sc/main/ipc.c b/drivers/soc/imx/sc/main/ipc.c
index 65af00b..fd888a0 100644
--- a/drivers/soc/imx/sc/main/ipc.c
+++ b/drivers/soc/imx/sc/main/ipc.c
@@ -344,10 +344,6 @@ int __init imx8_mu_init(void)
 					irq, err);
 			return err;
 		}
-
-		err = irq_set_irq_wake(irq, 1);
-		if (err)
-			pr_err("imx8mu_init: set_irq_wake failed: %d\n", err);
 	}
 
 	if (!scu_mu_init) {
-- 
1.7.9.5

